DentOS_Framework/DentOsTestbed/src/dent_os_testbed/test/test_suite/functional/L1/test_l1_mixed_speed.py

```diff
@@ -0,0 +1,156 @@
+import pytest, asyncio
+
+from dent_os_testbed.lib.ip.ip_link import IpLink
+from dent_os_testbed.lib.ethtool.ethtool import Ethtool
+
+from dent_os_testbed.utils.test_utils.tgen_utils import (
+    tgen_utils_get_dent_devices_with_tgen,
+    tgen_utils_traffic_generator_connect,
+    tgen_utils_dev_groups_from_config,
+    tgen_utils_get_traffic_stats,
+    tgen_utils_update_l1_config,
+    tgen_utils_setup_streams,
+    tgen_utils_start_traffic,
+    tgen_utils_stop_traffic,
+    tgen_utils_get_loss
+)
+
+from dent_os_testbed.utils.test_utils.tb_utils import tb_get_qualified_ports
+
+pytestmark = [pytest.mark.suite_functional_l1,
+              pytest.mark.asyncio,
+              pytest.mark.usefixtures("cleanup_bridges", "cleanup_tgen")]
+
+@pytest.mark.parametrize("speed, duplex, speed_ixia, duplex_ixia",
+                         [
+                            pytest.param(10, "full", 10, "half"),
+                            pytest.param(10, "full", 100, "full"),
+                            pytest.param(10, "full", 100, "half"),
+                            pytest.param(10, "full", 1000, "full"),
+
+                            pytest.param(10, "half", 10, "full"),
+                            pytest.param(10, "half", 100, "full"),
+                            pytest.param(10, "half", 100, "half"),
+                            pytest.param(10, "half", 1000, "full"),
+
+                            pytest.param(100, "full", 10, "full"),
+                            pytest.param(100, "full", 10, "half"),
+                            pytest.param(100, "full", 100, "half"),
+                            pytest.param(100, "full", 1000, "full"),
+
+                            pytest.param(100, "half", 10, "full"),
+                            pytest.param(100, "half", 10, "half"),
+                            pytest.param(100, "half", 100, "full"),
+                            pytest.param(100, "half", 1000, "full"),
+
+                            pytest.param(1000, "full", 10, "full"),
+                            pytest.param(1000, "full", 10, "half"),
+                            pytest.param(1000, "full", 100, "full"),
+                            pytest.param(1000, "full", 100, "half")
```

Comment:
This will produce a lot of tests and each of them will perform steps to create bridge, add ports, etc. Try to convert it to one test with multiple configurations:
1. init bridge
2. add ports, set to up state
3. create streams
4. set 1st L1 config combination, send and verify traffic
5. set 2nd L1 config combination, send and verify traffic
6. ...